azurerm/resource_arm_container_group.go

```diff
@@ -1002,6 +1235,56 @@ func flattenContainerVolumes(volumeMounts *[]containerinstance.VolumeMount, cont
 	return volumeConfigs
 }
 
+func flattenContainerProbes(input *containerinstance.ContainerProbe) []interface{} {
+	outputs := make([]interface{}, 0)
+	if input == nil {
+		return outputs
+	}
+
+	output := make(map[string]interface{})
+
+	if Exec := input.Exec; Exec != nil {
+		var exec []string
+		for _, v := range *input.Exec.Command {
+			exec = append(exec, v)
+		}
+		output["exec"] = exec
+	}
+
+	if HTTPGet := input.HTTPGet; HTTPGet != nil {
+		httpget := make(map[string]interface{})
+		httpget["path"] = *input.HTTPGet.Path
+		httpget["port"] = *input.HTTPGet.Port
+		httpget["scheme"] = input.HTTPGet.Scheme
+
+		output["httpget"] = []interface{}{httpget}
+
+	}
+
+	if FailureThreshold := input.FailureThreshold; FailureThreshold != nil {
+		output["failure_threashold"] = input.FailureThreshold
```

Comment:
This should be be `*FailureThreshold`, and with above suggestion:
```suggestion
		output["failure_threashold"] = *v
```